x/auth/keeper/deterministic_test.go

```diff
@@ -2,29 +2,52 @@ package keeper_test
 
 import (
 	"encoding/hex"
+	"sort"
 	"testing"
 
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"pgregory.net/rapid"
+
 	"github.com/cosmos/cosmos-sdk/baseapp"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	"github.com/cosmos/cosmos-sdk/testutil"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/auth/keeper"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/stretchr/testify/suite"
-	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
-	"pgregory.net/rapid"
 )
 
 type DeterministicTestSuite struct {
 	suite.Suite
 
+	key           *storetypes.KVStoreKey
 	ctx           sdk.Context
 	queryClient   types.QueryClient
 	accountKeeper keeper.AccountKeeper
 	encCfg        moduletestutil.TestEncodingConfig
+	maccPerms     map[string][]string
+}
+
+var (
+	iterCount = 1000
+	addr      sdk.AccAddress
+	pub       []byte
+)
+
+func init() {
+	addr = sdk.MustAccAddressFromBech32("cosmos1j364pjm8jkxxmujj0vp2xjg0y7w8tyveuamfm6")
+	pubkey := "01090C02812F010C25200ED40E004105160196E801F70005070EA21603FF06001E"
+
+	var err error
+	pub, err = hex.DecodeString(pubkey)
+	if err != nil {
+		panic(err)
+	}
 }
```

Comment:
I would really try to avoid init calls where possible. We can set all of these directly:

```
var (
  addr = sdk.MustAccAddressFromBech32("cosmos1j364pjm8jkxxmujj0vp2xjg0y7w8tyveuamfm6")
  pubkey, _ = hex.DecodeString("01090C02812F010C25200ED40E004105160196E801F70005070EA21603FF06001E")
)
````